README.md

```diff
@@ -13,45 +13,175 @@ A minimalistic wrapper around React Native's AsyncStorage.
 npm install react-native-simple-store
 ```
 
-## [API Reference](docs/index.md)
+## Use In Project
+```
+import store from 'react-native-simple-store;
+```
 
 ## Example Usage
 
-```js
-import store from 'react-native-simple-store';
-
-store
-  .save('coffee', {
-    isAwesome: true
-  })
-  .then(() => store.get('coffee'))
-  .then(coffee => {
-    console.assert(coffee.isAwesome === true);
-  })
-  .then(() => store.update('coffee', {
-    isNotEssential: false
-  }))
-  .then(() => store.get('coffee'))
-  .then(coffee => {
-    console.assert(coffee.isNotEssential === false);
-    console.assert(coffee.isAwesome === true);
-    return store.delete('coffee');
-  })
-  .then(() => store.get('coffee'))
-  .then(coffee => {
-    console.assert(coffee === null);
-  })
-  .catch(error => {
-    console.error(error.message);
-  });
-
-// using the .push method for storing arrays
-store
-  .save('coffeeTraits', ['rich'])
-  .then(store.push('coffeeTraits', 'smooth'))
-  .then(store.get('coffeeTraits'))
-  .then(console.log) // ['rich', 'smooth']
-```
+
+----------
+
+ - [Basic Object Handling](#objects)
+ - [Basic Array Handling](#arrays)
+ - [Chaining Methods/Error Handling](#chaining)
+ - [Deleting](#delete)
+
+
+<a name="objects"></a>
```

Comment:
Test this out, but I think that Github injects anchors with headings automatically. I think the link for this heading would be `#working-with-objects`. If that would work we could clean this up and not have to worry about adding in the anchors manually.